src/toolkits/object_detection/object_detector.cpp

```diff
@@ -1157,4 +1157,4 @@ void object_detector::update_model_metrics(gl_sframe data,
 }
 
 }  // object_detection
-}  // turi
+}  // namespace turi
```

Comment:
let's make 1159 and 1160 consistent in terms of whether to include "namespace".

also github reminds you to leave a blank line at the end of the file.